source/sphinx_extensions/command_block/templates/download.html

```diff
@@ -0,0 +1,53 @@
+<div class="download panel panel-default">
+  <div class="panel-heading white-bg">
+    Please select a download option that is most appropriate for your environment:
+  </div>
+
+  <div class="panel-body">
+    <ul class="nav nav-tabs gray-bg" role="tablist">
+      <li role="presentation" class="active">
+        <a href="#browser-{{ node.id }}" aria-controls="browser-{{ node.id }}" role="tab" data-toggle="tab">
+          Browser
+        </a>
+      </li>
+      <li role="presentation">
+        <a href="#wget-{{ node.id }}" aria-controls="wget-{{ node.id }}" role="tab" data-toggle="tab">
+          wget
+        </a>
+      </li>
+      <li role="presentation">
+        <a href="#curl-{{ node.id }}" aria-controls="messages-{{ node.id }}" role="tab" data-toggle="tab">
+          curl
+        </a>
+      </li>
+    </ul>
+
+    <div class="tab-content">
+      <div role="tabpanel" class="tab-pane fade in active" id="browser-{{ node.id }}">
+        <p>
+          <strong>Download URL:</strong> <a href="{{ node.url }}">
+            {{ node.url }}
+          </a>
+        </p>
+        <p>
+          <strong>Save as:</strong> {{ node.rename }}
```

Comment:
I think this should be `node.saveas` (it's used in two other places in this file as well).